### ISSUE TITLE
RxJava2?

### ISSUE THREAD

**Issue by sgrekov** [airbnb/RxGroups#34]
Hi!
Do you have any plans for migrating or any guides for using it with RxJava2?
Since RxJava2 released 3 months ago, I think it soon  will be a major blocker for using this library.

**Comment by felipecsl** [comment]
Now that Retrofit 2.2.0 is out with first party support for RxJava 2, I think we can finally upgrade. 
So yes, this should happen soon!

**Comment by sgrekov** [comment]
Cool! Thanks!

**Comment by VisheshVadhera** [comment]
@felipecsl Hey are you guys considering PRs for this one? I would love to move this project to RxJava 2.

**Comment by rahulrj** [comment]
Hi @felipecsl  i am working on migrating this library to RxJava2. Do you want to give any suggestions/ pointers for it?

**Comment by elihart** [comment]
@BenSchwab started work on the RxJava2 migration I believe.

**Comment by elihart** [comment]
There is quite a lot of new work in the 1.0 branch and the migration will probably happen there. not sure if it is worth making changes to master at this point.

**Comment by BenSchwab** [comment]
Yes, working on a RxJava2 migration now. I would prefer to keep the changes in the 1.0 branch. If you have started to think about the migration, would appreciate your feedback when I put up the PR @rahulrj

**Comment by BenSchwab** [comment]
https://github.com/airbnb/RxGroups/pull/42

**Comment by rahulrj** [comment]
@BenSchwab how is the code in `master` branch different from the `1.0` branch? Is `1.0` the active development branch and will it eventually get merged to `master`?

**Comment by BenSchwab** [comment]
Yep, 1.0 is the active development branch for the time being, with experimental features like annotation process based tagging - and now RxJava2. We are testing internally (and the branch may experience API churn) before elevating to a general 1.0 release.

**Comment by rahulrj** [comment]
Okay @BenSchwab . I was also working parallely on the RxJava2 migration. But i converted the code of the `master` branch.  Because the name `1.0` looked like that might be some old 1.0 release.

**Comment by meierjan** [comment]
any updates here? :)

**Comment by JEuler** [comment]
Hi! Any updates for rxjava 2.0 support?
